src/subdomains/core/referral/reward/ref-reward-dex.service.ts

```diff
@@ -39,14 +41,24 @@ export class RefRewardDexService {
       try {
         const asset = await this.assetService.getNativeAsset(blockchain);
 
+        // PayoutAsset Price
+        const assetPrice = await this.priceService.getPrice({
+          context: PriceRequestContext.REF_REWARD,
+          correlationId: `Ref reward ${new Date().toISOString()}`,
```

Comment:
`Ref reward` in correlation ID doesn't make much sense (context is already ref reward). But I forgot the blockchain, so I would use 
```ts
correlationId: `${blockchain}&${Util.isoDate(new Date())}`, 
```